code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -79,38 +79,37 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/venus_human_trap/handle_automated_action()
-	if(..())
```

Comment:
Uh, yes, it can

![img](https://i.gyazo.com/c4bf9cf6d6ae3d6ca17cdaf3bc7b0e33.png)